Makefile

```diff
@@ -21,16 +21,16 @@ FISHINSTALLDIR=${PREFIX}/share/fish/vendor_completions.d
 
 GO ?= go
 GOBIN := $(shell $(GO) env GOBIN)
-GOOS ?= $(shell go env GOOS)
-GOARCH ?= $(shell go env GOARCH)
+GOOS ?= $(shell $(GO) env GOOS)
+GOARCH ?= $(shell $(GO) env GOARCH)
 
 # N/B: This value is managed by Renovate, manual changes are
 # possible, as long as they don't disturb the formatting
 # (i.e. DO NOT ADD A 'v' prefix!)
 GOLANGCI_LINT_VERSION := 1.61.0
 
 ifeq ($(GOBIN),)
-GOBIN := $(GOPATH)/bin
+  GOBIN := $(subst \,/,$(shell $(GO) env GOPATH))/bin
```

Comment:
This is surprising to me. `make` doesn’t like native-style paths? Which implementation is this?

Reply:
On Windows, GOPATH include backslash not slash.